docs/src/content/docs/get-started/realtime-leaderboard.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: "Building a Realtime Leaderboard with DiceDB"
+description: "Create a gaming leaderboard in the easiest way possible."
+sidebar:
+  order: 4
+---
+
+In the world of gaming, leaderboards are a critical part of many real-time systems and are essential for tracking player
+rankings and improving engagement.
+DiceDB is a truly reactive database which allows you to eradicate the need to poll the database for changes by allowing clients to subscribe to changes in a sorted set (like a leaderboard).
+thus making it an excellent fit for implementing leaderboards.
+The goal of this example is to build a real-time leaderboard with DiceDB. We'll walk through the process of creating a gaming leaderboard using sorted set commands and our DiceDB SDK.
+
+But, before we start, make sure you have
+
+1. Go installed (at least version 1.18)
+2. Running instance of DiceDB
+3. Basic familiarity with [DiceDB and its CLI](https://github.com/dicedb/dice?tab=readme-ov-file#dice-in-action)
```

Comment:
This does not take to that particular section, please confirm if this is expected.